src/widgets/latest-blocks/LatestBlocks.vue

```diff
@@ -22,9 +25,8 @@
 
             <div class="latest-blocks__time">
               <TimeIcon class="latest-blocks__time-icon" />
-              {{ $t('time.min', [elapsed.allMinutes(block.created_at)]) }}
-              {{ $t('time.sec', [elapsed.seconds(block.created_at)]) }}
-              {{ $t('time.ago') }}
+              <TimeStamp v-bind="getTimeAgo(block.created_at)" />
```

Comment:
```suggestion
              <TimeAgo :value="block.created_at" />
```

That would be idiomatic.